script.js

```diff
@@ -0,0 +1,229 @@
+  KeyT: new Button(KEYBOARD, 'KeyT', 't', 'T', 'е', 'Е'),
+  KeyY: new Button(KEYBOARD, 'KeyY', 'y', 'Y', 'н', 'Н'),
+  KeyU: new Button(KEYBOARD, 'KeyU', 'u', 'U', 'г', 'Г'),
+  KeyI: new Button(KEYBOARD, 'KeyI', 'i', 'I', 'ш', 'Ш'),
+  KeyO: new Button(KEYBOARD, 'KeyO', 'o', 'O', 'щ', 'Щ'),
+  KeyP: new Button(KEYBOARD, 'KeyP', 'p', 'P', 'з', 'З'),
+  BracketLeft: new Button(KEYBOARD, 'BracketLeft', '[', '{', 'х', 'Х'),
+  BracketRight: new Button(KEYBOARD, 'BracketRight', ']', '}', 'ъ', 'Ъ'),
+  Delete: new Button(KEYBOARD, 'Delete', 'Delete', 'Delete', 'Delete', 'Delete'),
+  // row 3
+  CapsLock: new Button(KEYBOARD, 'CapsLock', 'CapsLock', 'CapsLock', 'CapsLock', 'CapsLock'),
+  KeyA: new Button(KEYBOARD, 'KeyA', 'a', 'A', 'ф', 'Ф'),
+  KeyS: new Button(KEYBOARD, 'KeyS', 's', 'S', 'ы', 'Ы'),
+  KeyD: new Button(KEYBOARD, 'KeyD', 'd', 'D', 'в', 'В'),
+  KeyF: new Button(KEYBOARD, 'KeyF', 'f', 'F', 'а', 'А'),
+  KeyG: new Button(KEYBOARD, 'KeyG', 'g', 'G', 'п', 'П'),
+  KeyH: new Button(KEYBOARD, 'KeyH', 'h', 'H', 'р', 'Р'),
+  KeyJ: new Button(KEYBOARD, 'KeyJ', 'j', 'J', 'о', 'О'),
+  KeyK: new Button(KEYBOARD, 'KeyK', 'k', 'K', 'л', 'Л'),
+  KeyL: new Button(KEYBOARD, 'KeyL', 'l', 'L', 'д', 'Д'),
+  Semicolon: new Button(KEYBOARD, 'Semicolon', ';', ':', 'ж', 'Ж'),
+  Quote: new Button(KEYBOARD, 'Quote', '\'', '"', 'э', 'Э'),
+  Enter: new Button(KEYBOARD, 'Enter', 'Enter', 'Enter', 'Enter', 'Enter'),
+  // row 4
+  ShiftLeft: new Button(KEYBOARD, 'ShiftLeft', 'Shift', 'Shift', 'Shift', 'Shift'),
+  KeyZ: new Button(KEYBOARD, 'KeyZ', 'z', 'Z', 'я', 'Я'),
+  KeyX: new Button(KEYBOARD, 'KeyX', 'x', 'X', 'ч', 'Ч'),
+  KeyC: new Button(KEYBOARD, 'KeyC', 'c', 'C', 'с', 'С'),
+  KeyV: new Button(KEYBOARD, 'KeyV', 'v', 'V', 'м', 'М'),
+  KeyB: new Button(KEYBOARD, 'KeyB', 'b', 'B', 'и', 'И'),
+  KeyN: new Button(KEYBOARD, 'KeyN', 'n', 'N', 'т', 'Т'),
+  KeyM: new Button(KEYBOARD, 'KeyM', 'm', 'M', 'ь', 'Ь'),
+  Comma: new Button(KEYBOARD, 'Comma', ',', '<', 'б', 'Б'),
+  Period: new Button(KEYBOARD, 'Period', '.', '>', 'ю', 'Ю'),
+  Slash: new Button(KEYBOARD, 'Slash', '/', '?', '.', ','),
+  ArrowUp: new Button(KEYBOARD, 'ArrowUp', '↑', '↑', '↑', '↑'),
+  ShiftRight: new Button(KEYBOARD, 'ShiftRight', 'Shift', 'Shift', 'Shift', 'Shift'),
+  // row 4
+  ControlLeft: new Button(KEYBOARD, 'ControlLeft', 'Ctrl', 'Ctrl', 'Ctrl', 'Ctrl'),
+  AltLeft: new Button(KEYBOARD, 'AltLeft', 'Alt', 'Alt', 'Alt', 'Alt'),
+  Space: new Button(KEYBOARD, 'Space', 'Space', 'Space', 'Space', 'Space'),
+  AltRight: new Button(KEYBOARD, 'AltRight', 'Alt', 'Alt', 'Alt', 'Alt'),
+  ArrowLeft: new Button(KEYBOARD, 'ArrowLeft', '←', '←', '←', '←'),
+  ArrowDown: new Button(KEYBOARD, 'ArrowDown', '↓', '↓', '↓', '↓'),
+  ArrowRight: new Button(KEYBOARD, 'ArrowRight', '→', '→', '→', '→'),
+  ControlRight: new Button(KEYBOARD, 'ControlRight', 'Ctrl', 'Ctrl', 'Ctrl', 'Ctrl'),
+};
+
+function changeLanguage() {
+  if (language === 'en') language = 'ru';
```

Comment:
it is better to use brackts for if else blocks
```
if (language === 'en'){
    language = 'ru';
}  else {
   language = 'en'
}
```
or even ternary in this case
`language = language === 'en' ? 'ru' : 'en'`

Reply:
fix in commit 6597863